### ISSUE TITLE
When we use the pulsar-admin.sh script to delete partitioned topic, it will throw the http error, error code is 500

### ISSUE THREAD

**Issue by yiyi1223** [apache/pulsar#5558]
**Describe the bug**
When we use the pulsar-admin.sh script to delete partitioned topic, it will throw the http error, error code is 500.

**To Reproduce**
1. when we run the benchmark test, the cluster of bookie has one or more bookie state that is read only;
2. and then we use the script to delete the partitioned topic, ../bin/pulsar-admin topics deleted-partition-topic persistent://tenant/namespace/topic;
3. we receive the http error, error code is 500, we can't delete successfully;
4. we restart all pulsar brokers of cluster,and then run the script to delete, it succeed.

**Expected behavior**
succeed to delete the partitioned topic.

**Screenshots**
we check the service log. find the error as follow:
```
15:15:29.005 [bookkeeper-ml-workers-OrderedExecutor-7-0] ERROR org.apache.bookkeeper.common.util.SafeRunnable - Unexpected throwable caught
java.lang.IllegalArgumentException: inconsistent range
        at java.util.concurrent.ConcurrentSkipListMap$SubMap.<init>(ConcurrentSkipListMap.java:2620) ~[?:1.8.0_171]
        at java.util.concurrent.ConcurrentSkipListMap.subMap(ConcurrentSkipListMap.java:2078) ~[?:1.8.0_171]
        at org.apache.bookkeeper.mledger.util.RangeCache.removeRange(RangeCache.java:132) ~[org.apache.pulsar-managed-ledger-2.5.0-d9ebec524.jar:2.5.0-d9ebec524]
        at org.apache.bookkeeper.mledger.impl.EntryCacheImpl.invalidateEntries(EntryCacheImpl.java:151) ~[org.apache.pulsar-managed-ledger-2.5.0-d9ebec524.jar:2.5.0-d9ebec524]
        at org.apache.bookkeeper.mledger.impl.ManagedLedgerImpl$7.operationComplete(ManagedLedgerImpl.java:779) ~[org.apache.pulsar-managed-ledger-2.5.0-d9ebec524.jar:2.5.0-d9ebec524]
        at org.apache.bookkeeper.mledger.impl.ManagedLedgerImpl$7.operationComplete(ManagedLedgerImpl.java:767) ~[org.apache.pulsar-managed-ledger-2.5.0-d9ebec524.jar:2.5.0-d9ebec524]
        at org.apache.bookkeeper.mledger.impl.MetaStoreImplZookeeper.lambda$null$13(MetaStoreImplZookeeper.java:308) ~[org.apache.pulsar-managed-ledger-2.5.0-d9ebec524.jar:2.5.0-d9ebec524]
        at org.apache.bookkeeper.mledger.util.SafeRun$1.safeRun(SafeRun.java:32) ~[org.apache.pulsar-managed-ledger-2.5.0-d9ebec524.jar:2.5.0-d9ebec524]
        at org.apache.bookkeeper.common.util.SafeRunnable.run(SafeRunnable.java:36) [org.apache.bookkeeper-bookkeeper-common-4.9.2.jar:4.9.2]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_171]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_171]
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [io.netty-netty-common-4.1.43.Final.jar:4.1.43.Final]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
```

**Desktop (please complete the following information):**

**Additional context**

**Comment by Jennifer88huang** [comment]
@codelipenghui @tuteng Could you please help check the issue and give explanation? Thank you.

**Comment by jiazhai** [comment]
need handling the exception reported

**Comment by sijie** [comment]
The root cause of this problem will be fixed in #5621
